boards.txt

```diff
@@ -1,20 +1,20 @@
 menu.version=Version
 
 iskra-nano-pb.name=Amperka Iskra Nano PB
-iskra-nano-pb.upload.tool=avrdudecustom
+iskra-nano-pb.upload.tool=arduino:avrdude
 iskra-nano-pb.upload.protocol=arduino
 iskra-nano-pb.upload.mcu=atmega328pb
 iskra-nano-pb.upload.maximum_size=32256
 iskra-nano-pb.upload.maximum_data_size=2048
 
-iskra-nano-pb.bootloader.tool=avrdudecustom
+iskra-nano-pb.bootloader.tool=arduino:avrdude
 iskra-nano-pb.bootloader.low_fuses=0xFF
 iskra-nano-pb.bootloader.high_fuses=0xDE
 iskra-nano-pb.bootloader.unlock_bits=0xFF
 iskra-nano-pb.bootloader.lock_bits=0xCF
 
-iskra-nano-pb.build.mcu=atmega328p
-iskra-nano-pb.build.board=AVR_A_STAR_328PB
+iskra-nano-pb.build.mcu=atmega328pb
+iskra-nano-pb.build.board=AVR_ISKRA-NANO-PB
```

Comment:
If I understand correctly this string is `#define`’d while compiling. `AVR_ISKRA-NANO-PB` is not a valid C++ identifier, so should be `AVR_ISKRA_NANO_PB`

Reply:
Fixed. Funny thing is that `AVR_ISKRA-NANO-PB` is generated by `arduino-cli` :)